### PR TITLE
Blog - This Week in Lotus

### PR DESCRIPTION
**Pull request by rjan90**
Add This Week in Lotus in the blog-section. Replaces #304

### CODE REVIEW

**Comment by johnnymatthews** [comment]
Is this a work in progress? There's nothing in the topbar nav.

**Comment by rjan90** [comment]
> Is this a work in progress? There's nothing in the topbar nav.
I think this is ready for review now, as it just adds all the historical posts to the blog-page. With regards to where to expose a link to the blog-section, we have to think a bit further about that 😄

File: content/en/blog/_index.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: "Blog"
-description: "The Doks Blog."
+description: "The Lotus Blog."
 date: 2020-10-06T08:49:55+00:00
 lastmod: 2020-10-06T08:49:55+00:00
```
Comment (johnnymatthews) [inline content/en/blog/_index.md:5]:
```suggestion
```

**Review by johnnymatthews — APPROVED** [review]
Sounds good!

**Pull request merged**